lib/cc/services/github_pull_requests.rb

```diff
@@ -87,8 +86,7 @@ def presenter
   def update_status_error
     update_status(
       "error",
-      "Code Climate encountered an error while attempting to analyze this " +
-        "pull request."
+      message
```

Comment:
Mmmm makes sense. 

Imagine we'll probably also want to avoid fetching the `message` then.